Automattic/jetpack
# ThreatsDataViews: Update isPrimary actions

Description #
Moves access to actions toggles inline and hides the collapsible actions list

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1472

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Load storybook with `cd projects/js-packages/storybook && npm run storybook:dev`
* View the Threats Data View component
* Ensure the content of the actions section is positioned and styled according to designs in both Table and List views
* Review the code and ensure the implementation is correct

## Screenshots
Table View:
![Screen Shot on 2024-10-22 at 13-57-16](https://github.com/user-attachments/assets/1cbb74b4-59dd-4fd0-b38e-5950784c63f7)

List View:
![Screen Shot 2024-10-22 at 14 32 29](https://github.com/user-attachments/assets/bac1c003-d0d7-486d-a84e-91ab8dea7323)
